### PR TITLE
27 css header

### PR DESCRIPTION
**Pull request by poshivkin**

### CODE REVIEW

File: css/burger.css [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+  display: none;
+}
+
+.burger.active::after {
+  bottom: 50%;
+  transform: translateY(50%) rotate(-45deg);
+}
+
+.burger::before {
+  content: "";
```
Comment (Maiia2023) [inline css/burger.css]: better to use single quotes
add "display: block;"

File: css/burger.css [hunk 2]
```diff
@@ -1,5 +1,48 @@
...
+  width: 100%;
+  height: 2px;
+  background: red;
+  position: absolute;
+  top: 0;
+  transform: rotate(45deg);
```
Comment (Maiia2023) [inline css/burger.css]: delete "transform: rotate(45deg);"

File: css/burger.css [hunk 3]
```diff
@@ -1,5 +1,48 @@
...
+
+.burger span {
+  display: block;
+  width: 100%;
+  height: 2px;
+  background: yellow;
+}
+
+.burger::after {
+  content: "";
```
Comment (Maiia2023) [inline css/burger.css]: better to use single quotes
add "display: block;"

File: css/burger.css [hunk 4]
```diff
@@ -1,5 +1,48 @@
...
+  transition: all 2s;
+}
+
+.burger.active span {
+  display: none;
+}
+
+.burger.active::after {
+  bottom: 50%;
+  transform: translateY(50%) rotate(-45deg);
```
Comment (Maiia2023) [inline css/burger.css:19]: add "transition: all 2s;"

File: index.html [hunk 5]
```diff
@@ -238,24 +268,38 @@ <h3 class="column__title">Contact</h3>
           <ul class="column__list">
             <!-- Item 1 -->
             <li class="column__item">
-              <a class="footer__logo" href="mailto:hello@zen.com">hello@zen.com</a>
+              <a class="footer__logo" href="mailto:hello@zen.com"
+                >hello@zen.com</a
+              >
```
Comment (Maiia2023) [inline index.html:274]: in one line looks better:
```
<a class="footer__logo" href="mailto:hello@zen.com">hello@zen.com</a>
```
or like this:
```
<a class="footer__logo" href="mailto:hello@zen.com">
    hello@zen.com
</a>
```

File: index.html [hunk 6]
```diff
@@ -295,8 +339,10 @@ <h3 class="modal__title">Order a consultation</h3>
           </label>
           <!-- Privacy policy -->
           <label class="form__checkbox">
-            <input type="checkbox" id="policy"/>
-            <a href="https://example.com/policy" target="_blank">Privacy policy</a> 
+            <input type="checkbox" id="policy" />
+            <a href="https://example.com/policy" target="_blank"
+              >Privacy policy</a
+            >
```
Comment (Maiia2023) [inline index.html:346]:
```
<a href="https://example.com/policy" target="_blank">Privacy policy</a>
```

File: css/burger.css (hunk 1 above)
Comment (poshivkin) [reply css/burger.css]: > better to use single quotes add "display: block;"

why is it better to use single quotes marks ?

File: css/header.css [hunk 7]
```diff
@@ -1,6 +1,7 @@
 .header {
   padding: 10px 15px;
   border-bottom: 1px solid #aaa;
+  margin-bottom: 100vh;
```
Comment (Maiia2023) [inline css/header.css]: remove this line

File: css/burger.css (hunk 1 above)
Comment (Maiia2023) [reply css/burger.css]: It's just a matter of coding style (I can only recommend stick to the mentor's style, but it's up to you)

**Review by Maiia2023 — APPROVED** [review]

**Review by Maiia2023 — APPROVED** [review]

**Review by Maiia2023 — APPROVED** [review]

**Review by Maiia2023 — APPROVED** [review]

**Review by Maiia2023 — APPROVED** [review]

**Review by Maiia2023 — APPROVED** [review]

**Review by Maiia2023 — APPROVED** [review]

**Pull request merged**